RPi-Pico/config/user_settings.h

```diff
@@ -27,12 +27,15 @@
 #ifndef WOLFSSL_USER_SETTINGS_H
 #define WOLFSSL_USER_SETTINGS_H
 
+#define WOLFSSL_AUTOSAR
```

Comment:
Defaulting to have WOLFSSL_AUTOSAR on here is odd? Why have it enabled? The other time changes in this pull request look good though.